packages/bot-web-ui/src/components/run-panel.jsx

```diff
@@ -98,6 +98,54 @@ const drawerFooter = ({
     );
 };
 
+const mobileDrawerFooter = ({
```

Comment:
Since this is a component we can capitalise the first character too, i.e. `MobileDrawerFooter`